sdk/push/src/main/java/com/klaviyo/push/KlaviyoPushService.kt

```diff
@@ -41,11 +95,11 @@ class KlaviyoPushService : FirebaseMessagingService() {
      */
     override fun onNewToken(newToken: String) {
         super.onNewToken(newToken)
+        setPushToken(newToken)
+    }
 
-        val properties = KlaviyoCustomerProperties()
-            .addAppendProperty(REQUEST_PUSH_KEY, newToken)
-        Klaviyo.identify(properties)
-
-        KlaviyoPreferenceUtils.writeStringPreference(PUSH_TOKEN_PREFERENCE_KEY, newToken)
+    override fun onMessageReceived(message: RemoteMessage) {
```

Comment:
docstring here too plz. We'll likely build source docs off of these at some point as a development aid for engineers so would like to keep every function documented